### PR TITLE
🐛 Fix forbidden terms to unblock `main`

### PR DESCRIPTION
**Pull request by danielrozenberg**

### CODE REVIEW

**Review by jridgewell — APPROVED** [review]

**Review by rcebulko — APPROVED** [review]

File: build-system/test-configs/forbidden-terms.js [hunk 1]
```diff
@@ -932,7 +931,6 @@ const forbiddenTermsSrcInclusive = {
       'ads/_a4a-config.js',
       'build-system/server/amp4test.js',
       'build-system/server/app-index/amphtml-helpers.js',
-      'build-system/server/app-utils.js',
```
Comment (rcebulko) [inline build-system/test-configs/forbidden-terms.js:935]: Was this reverted line intentional?

Comment (danielrozenberg) [reply build-system/test-configs/forbidden-terms.js:935]: Yes, #34754 removed direct mentions to `cdn.ampproject` in that file (replacing it with a regex, so now it's `cdn\\.ampproject` instead) and that also broke the checks

File: build-system/test-configs/forbidden-terms.js [hunk 2]
```diff
@@ -605,21 +605,20 @@ const forbiddenTermsGlobal = {
...
+  '\\b__AMP_EXP\\b': {
     message:
       'Do not access __AMP_EXP directly. Use isExperimentOn() to access config',
     allowlist: [
       'src/experiments/index.js',
       'src/experiments/experiments.extern.js',
     ],
   },
-  'AMP_EXP': {
+  '\\bAMP_EXP\\b': {
```
Comment (rcebulko) [inline build-system/test-configs/forbidden-terms.js:616]: nit for a follow-up: just one rule `'\\b(__)?AMP_EXP\\b'`

Comment (danielrozenberg) [reply build-system/test-configs/forbidden-terms.js:616]: Ugh this is _almost_ done passing the checks... wanna put that in your followup PR? :D

**Review by rileyajones — APPROVED** [review]

**Pull request merged**